1_turb_abl_fsi/template_files/nrel5mw_nalu.yaml

```diff
@@ -0,0 +1,384 @@
+      turbulent_ke: solve_scalar
+      specific_dissipation_rate: solve_scalar
+      pressure: solve_elliptic
+      ndtw: solve_elliptic
+
+    systems:
+    - WallDistance:
+        name: myNDTW
+        max_iterations: 1
+        convergence_tolerance: 1.0e-8
+        update_frequency: 1000000000
+        exchange_fringe_data: false
+
+    - LowMachEOM:
+        name: myLowMach
+        max_iterations: 1
+        convergence_tolerance: 1e-8
+
+    - ShearStressTransport:
+        name: mySST
+        max_iterations: 1
+        convergence_tolerance: 1e-8
+
+  initial_conditions:
+    - constant: ic_1
+      target_name:
+        - blade1-HEX
+        - blade2-HEX
+        - blade3-HEX
+{if(TOWER)}
+        - tower-fluid-HEX
+{endif}
+      value:
+        pressure: 0.0
+        velocity: [9.8726896031426, 5.7, 0.0]
+        turbulent_ke: 1.0
+        specific_dissipation_rate: 250.0
+
+  material_properties:
+    target_name:
+      - blade1-HEX
+      - blade2-HEX
+      - blade3-HEX
+{if(TOWER)}
+      - tower-fluid-HEX
+{endif}
+
+    specifications:
+    - name: density
+      type: constant
+      value: 1.0
+    - name: viscosity
+      type: constant
+      value: 1.8e-5
+
+  boundary_conditions:
+
+  - wall_boundary_condition: bc_wall
+    target_name: blade1
+    wall_user_data:
+      use_wall_function: no
+      turbulent_ke: 0.0
+
+  - wall_boundary_condition: bc_wall
+    target_name: blade2
+    wall_user_data:
+      use_wall_function: no
+      turbulent_ke: 0.0
+
+  - wall_boundary_condition: bc_wall
+    target_name: blade3
+    wall_user_data:
+      use_wall_function: no
+      turbulent_ke: 0.0
+
+{if(TOWER)}
+  - wall_boundary_condition: bc_wall
+    target_name: tower-wall
+    wall_user_data:
+      use_wall_function: no
+      turbulent_ke: 0.0
+
+  - symmetry_boundary_condition: ground
+    target_name: tower-ground
+    symmetry_user_data:
+{endif}
+
+  - overset_boundary_condition: bc_overset
+    overset_connectivity_type: tioga
+    overset_user_data:
+      tioga_populate_inactive_part: false
+      mesh_group:
+      - overset_name: Turbine
+        mesh_parts:
+          - blade1-HEX
+          - blade2-HEX
+          - blade3-HEX
+{if(TOWER)}
+          - tower-fluid-HEX
+{endif}
+        wall_parts: 
+          - blade1 
+          - blade2
+          - blade3
+{if(TOWER)}
+          - tower-wall
+{endif}
+        ovset_parts:
+          - outerBC
+{if(TOWER)}
+          - tower-overset
+{endif}
+
+  mesh_transformation:
+  - name: turbine_shaft_tilt
+    mesh_parts:
+      - blade1-HEX
+      - blade2-HEX
+      - blade3-HEX
+    motion:
+      - type: rotation
+        angle: 5.0
+        axis: [0.0, 1.0, 0.0]
+        centroid: [0.0, 0.0, 0.0]
+      - type: translation
+        displacement: [1772.54, 2080.0, 90.0]
+        
+{if(TOWER)}
+  - name: tower_translate
+     mesh_parts:
+      - tower-fluid-HEX
+    motion:
+      - type: translation
+        displacement: [1777.54, 2080.0, 90.0]
```

Comment:
@psakievich thanks, this looks good. As part of this could we also move this to `1800, 1800, 90.0` ? This non-rounded off positioning is bothering me a lot, and was used to be consistent with one of the initial setups. As we move to multiple turbines, it is easier to have whole numbers for the position. I will comment on other places too. We need to change this information in the `inp.yaml` too.